homeassistant/components/dnsip/sensor.py

```diff
@@ -53,12 +54,18 @@ async def async_setup_entry(
 
     resolver_ipv4 = entry.options[CONF_RESOLVER]
     resolver_ipv6 = entry.options[CONF_RESOLVER_IPV6]
+    port_ipv4 = entry.options[CONF_PORT]
+    port_ipv6 = entry.options[CONF_PORT_IPV6]
 
     entities = []
     if entry.data[CONF_IPV4]:
-        entities.append(WanIpSensor(name, hostname, resolver_ipv4, False))
+        entities.append(
+            WanIpSensor(name, hostname, resolver_ipv4, False, port=port_ipv4)
```

Comment:
```suggestion
            WanIpSensor(name, hostname, resolver_ipv4, False, port_ipv4)
```
A bit odd to use the parameter name only for the last one, also below for ipv6